techniques/system/common/1.0/rudder_lib.st

```diff
@@ -468,6 +468,16 @@ body perms mg(mode,group)
 }
 
 ########################################################################
+# Select a file using a pattern                                        #
+########################################################################
+
+body file_select simple_pattern(pattern)
```

Comment:
Fixed, also rebased to latest 2.11, which fixed the previously bad ${g.gzip} variable